Ternary Search Tree/TST.playground/Sources/TernarySearchTree.swift

```diff
@@ -17,12 +17,12 @@ public class TernarySearchTree<Element> {
 
     //MARK: - Insertion
 
-    public func insert(data: Element, withKey key: String) -> Bool {
+    public func insertData(_ data: Element, withKey key: String) -> Bool {
```

Comment:
Do you think the function would look cleaner if we did the following?
func insert(data: Element, withKey key: String) -> Bool  {}